src/stormpath.config.js

```diff
@@ -169,6 +186,24 @@ angular.module('stormpath.CONFIG',[])
     /**
     * @ngdoc property
     *
+    * @name OAUTH_PROVIDERS_ENDPOINT
+    *
+    * @propertyOf stormpath.STORMPATH_CONFIG:STORMPATH_CONFIG
+    *
+    * @description
+    *
+    * Default: `/oauth/provider`
+    *
+    * The endpoint that is used for fetching a list of available OAuth providers.
+    * Used by {@link stormpath.socialLoginService.$socialLogin#getProviders $socialLogin.getProviders()}.
+    *
+    */
+    OAUTH_PROVIDERS_ENDPOINT: '/oauth/providers',
```

Comment:
We want to change this URL, as the returned data structure doesn't actually have anything to do with OAuth authentication.  My suggestion is to rename this endpoint to `/spa-config` and have an object in it with a property of `socialProviders`.  Thoughts?